delivery-process/external-contributions.md

```diff
@@ -0,0 +1,75 @@
+# Working with External GitHub Contributions
+
+This document outlines various approaches for working with pull requests (PRs) from external contributors on GitHub.
+
+## Checking Out External Pull Requests
+
+### Using Git Commands
+
+You can directly pull external PRs with the following Git commands:
+
+```bash
+git fetch origin pull/<PR-ID>/head:<BRANCH_NAME>
+git switch <BRANCH_NAME>
+```
+
+Replace `<PR-ID>` with the PR number and `<BRANCH_NAME>` with your desired branch name.
+
+### Using GitHub CLI
+
+Install the GitHub CLI tool (`gh`) and use:
+
+```bash
+gh pr checkout <PR-ID>
+```
+
+This allows you to checkout PRs and push against them directly without configuring remotes.
+
+### Using IDE Integration
+
+Most modern IDEs have GitHub integrations that allows you to:
+- Browse all PRs
+- Review PR files
+- Checkout PRs with a single click
+
+## Pushing Changes to External PRs
+
+### Prerequisites
+
+To push changes to a PR from a fork, the contributor must have checked "Allow edits from maintainers" when creating the PR.
+
+To check if this option is enabled:
+1. Go to the "Files changed" tab in the PR
+2. Click the three dots menu next to a file
+3. If "Edit file" is grayed out, the option is not checked
+
+If the option is not enabled, ask the contributor to enable it.
+
+### Method 1: Checked out with GitHub CLI
+
+After checking out the PR with `gh pr checkout <PR-ID>`:
+1. Make your changes
+2. Commit your changes: `git add . && git commit -m "Your message"`
+3. Push using standard git commands: `git push`
+
+The GitHub CLI should automatically set the correct remote, so a simple push should work.
+
+### Method 2: Adding a Remote for the Fork
+
+If you need more control or are experiencing issues with the GitHub CLI:
+
+1. Add the fork's repository as a new remote:
+   ```bash
+   git remote add fork-owner git@github.com:fork-owner/repository-name.git
+   ```
+   Replace `fork-owner` with the GitHub username of the fork owner and `repository-name` with the name of the repository.
+
+2. Push your commits to this remote:
+   ```bash
+   git push fork-owner branch-name
+   ```
+
+## When You Can't Push to a PR
+
+If the contributor has not enabled "Allow edits from maintainers", ask them politely to enable this option.
+If they don't respond or refuse, you may need to create a new PR and close the old one (though this should be avoided if possible as it's less collaborative).
```

Comment:
Maybe we should explain that we should update the Author when you create a new PR.
There are also lovely tools like `lazygit` that you can use.

Reply:
added a sentence for that, please have a look